### PR TITLE
feat:update user center

### PR DESCRIPTION
**Pull request by MayueCif**
# Description

docker user center auth 

* [X] Code compiles correctly
* [X] Created/updated tests
* [X] Extended the documentation

### CODE REVIEW

File: src/Masa.Stack.Components/Layouts/Layout.razor [hunk 1]
```diff
@@ -7,36 +7,38 @@
...
-        <SearchMenu />
-        <MSpacer />
-        <Favorites FlattenedNavs="FlattenedNavs" />
-        <Languages />
-        <MSheet class="d-flex align-center justify-space-between rounded-6 pa-1 mr-n3" Width="100">
-            <Notification />
-            <User UserCenterUrl="@UserCenterRoute" />
-        </MSheet>
-    </MAppBar>
+<CascadingValue Value="this">
```
Comment (capdiem) [inline src/Masa.Stack.Components/Layouts/Layout.razor:10]: Use `IsFixed`.

File: src/Masa.Stack.Components/Layouts/Components/Modals/LogoutModal.razor [hunk 2]
```diff
@@ -8,6 +8,8 @@
 </ADialog>
 
 @code {
+    [CascadingParameter]
+    public Layout Root { get; set; } = null!;
```
Comment (capdiem) [inline src/Masa.Stack.Components/Layouts/Components/Modals/LogoutModal.razor:12]: Reanme to `Layout`?

File: src/Masa.Stack.Components/Layouts/Components/User.razor [hunk 3]
```diff
@@ -51,11 +52,24 @@
 <UpdatePasswordModal @ref="_updatePasswordModal"></UpdatePasswordModal>
 
 @code {
-
     [Parameter]
     public string? UserCenterUrl { get; set; }
 
     private LogoutModal _logoutModal = null!;
     private UpdatePasswordModal _updatePasswordModal = null!;
+    private string avatar = "./_content/Masa.Stack.Components/img/avatar/default.svg";
```
Comment (capdiem) [inline src/Masa.Stack.Components/Layouts/Components/User.razor:60]: CDN!

File: tests/MasaWebApp/Program.cs [hunk 4]
```diff
@@ -12,7 +12,8 @@
     options.UserName = "name";
     options.UserId = "sub";
 });
-builder.Services.AddMasaStackComponentsForServer("wwwroot/i18n", "https://auth-service-develop.masastack.com/");
+//builder.Services.AddMasaStackComponentsForServer("wwwroot/i18n", "https://auth-service-develop.masastack.com/");
+builder.Services.AddMasaStackComponentsForServer("wwwroot/i18n", "http://localhost:18002/");
```
Comment (capdiem) [inline tests/MasaWebApp/Program.cs:16]: ???

File: tests/MasaWebApp/Shared/MainLayout.razor [hunk 5]
```diff
@@ -1,8 +1,7 @@
 ﻿@using Masa.Stack.Components.Models
 @inherits LayoutComponentBase
 
-<Layout AppId="123"
-        UserCenterRoute="/user-center"
+<Layout AppId="123" SignOut="SignOut"
```
Comment (capdiem) [inline tests/MasaWebApp/Shared/MainLayout.razor:4]: Rename to `OnSignOut`

**Review by capdiem — CHANGES_REQUESTED** [review]

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**